### PR TITLE
Meta: remove zqzhang as reviewer

### PR DESCRIPTION
**Pull request by annevk**
They have not been active for years.

Also add myself to File API.

### CODE REVIEW

**Review by nt1m — APPROVED** [review]

File: webmessaging/META.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 spec: https://html.spec.whatwg.org/multipage/web-messaging.html
```
Comment (doruk5) [inline webmessaging/META.yml:1]: Zamani var

Comment (doruk5) [reply webmessaging/META.yml:1]: Zamani var

Comment (doruk5) [inline webmessaging/META.yml:1]: Zamani var

File: uievents/META.yml [hunk 2]
```diff
@@ -2,4 +2,3 @@ spec: https://w3c.github.io/uievents/
 suggested_reviewers:
```
Comment (doruk5) [inline uievents/META.yml:2]: Zamani var

Comment (doruk5) [reply uievents/META.yml:2]: Zamani yakin

Comment (doruk5) [reply uievents/META.yml:2]: Zaman yaklasiyor

**Review by doruk5 — COMMENTED** [review]
[redacted spam]

**Pull request merged**